Cargo.toml

```diff
@@ -11,11 +11,11 @@ homepage = "https://github.com/tailhook/serde-regex"
 documentation = "https://docs.rs/serde_regex"
 version = "0.3.1"
 authors = ["paul@colomiets.name"]
+edition = "2018"
 
 [dependencies]
-serde = "1.0.0"
+serde = { version = "1.0.0", features = ["derive"] }
```

Comment:
Why was this done? Previously, `serde_derive` was only a dev dependency, but this makes `serde_derive` a hard dependency and it doesn't look like it's needed?

Reply:
I agree with you. Serde derive macro are only used for doc test. I have revert dependencies to previous version.